vms/platformvm/state/state.go

```diff
@@ -420,6 +429,18 @@ type txAndStatus struct {
 	status status.Status
 }
 
+type wrappedFxOwner struct {
+	owner fx.Owner
+	size  int
+}
+
+func wrappedFxOwnerSize(_ ids.ID, w *wrappedFxOwner) int {
```

Comment:
If we change our cache to be of type `wrappedFxOwner` instead of `*wrappedFxOwner` I think we can simplify the below if to be the following:

```
return ids.IDLen + w.size
```

And make the invariant that if `wrappedFxOwner` is the zero value `wrappedFxOwner{}`, then the key is not in the database.

Reply:
Modified, good call.